.github/CODEOWNERS

```diff
@@ -0,0 +1,4 @@
+*    @gsganden
+.ci/*    @jameslamb
+.travis.yml    @jameslamb
+.github/*    @jameslamb
```

Comment:
I think we should both be on `.github/`, since you (as the primary maintainer) own the pull request template

Reply:
Good point. I changed it to just me. Is there a reason to add you there?

Reply:
oh nope I don't have to be there!